tests/run-tests

```diff
@@ -552,8 +552,10 @@ the last matching regex is used:
   run-tests -i async - exclude all, then include tests containg "async" anywhere
   run-tests -e '/big.+int' - include all, then exclude by regex
   run-tests -e async -i async_foo - include all, exclude async, yet still include async_foo
-Expectations for tests that do not work in CPython can be saved in a .exp file. That file can be generated by
-running the test without .exp file and saving the .out file produced as a result of the failing test.
+Run-tests compares the MicroPython output of the test with expectations produced by running the test
```

Comment:
Using a captical for `run-tests` here could cause confusion due to the case-sensitive nature of some filesystems. Also I'd change `expectations produced` into 'output produced': expectations is inconsistent with 'MicroPython output' and sounds like something special while it's just the 'CPython output'.

But all of this is only correct when run-tests is effectively running tests and not one of the other modes so maybe that should be mentioned explicitly.